src/Table2Beta/SelectedRowsHeader.tsx

```diff
@@ -68,7 +68,7 @@ export default function SelectedRowsHeader({
               </div>
             </>
           )}
-          {allSelected && (
+          {rowsAreSelected && allSelected && (
```

Comment:
when would `rowsAreSelected = false` but `allSelected = true` - naming at least seems redundant, but I also am not familiar enough to know for sure

Reply:
Good question! This PR actually addresses a situation where `rowsAreSelected=false` but `allSelected = true`. To be fair, that scenario only occurred because I forgot to update `allSelected` 😛 
We use `allSelected` because SelectedRowsHeader does not know how many rows there are total. So we either 1) pass in the total number of rows available for selection, or 2) pass `allSelected`.